### DISCUSSION TITLE
Issue with NextAuth integration and token expiration on Supabase DB

### DISCUSSION THREAD

**Discussion by meezin** [supabase/supabase#26021] [Questions]
I am encountering an issue with token expiration while using NextAuth for OAuth operations with Figma and storing access tokens in a Supabase database.

Previously, I was utilizing a Heroku database without any issues, where access tokens received via Figma OAuth2 had a default expiration of 90 days. However, upon migrating to Supabase, I noticed that attempting to use the access token stored in the database table within less than a day results in receiving an `invalid token 403 response`

I haven't made any adjustments to the user table where the access token is stored in the authentication configuration of Supabase, and I have not separately configured it for Supabase Auth. Could this lack of specific configuration potentially be causing the issue?

The supabase is triggered as a auth even I don't even use their auth system.. 

![image](https://github.com/supabase/supabase/assets/18614465/7164d7c9-8284-4f8a-b51c-4886b1fb68d4)

Any insights or suggestions would be greatly appreciated.

**Comment by GaryAustin1** [comment]
The auth events you see are probably health check when using the dashboard.  You can look at the API report (under Reports) and see.

If you are using jwt's from another source (minted with the Supabase JWT secret) they should be good until they expire (exp claim in the jwt).
Supabase would not be touching the data stored in your table or  modifying your JWTs.
